core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2Spec.scala

```diff
@@ -415,8 +410,8 @@ class BillingApiServiceV2Spec extends ApiServiceSpec with MockitoSugar {
   it should "return 400 when creating a project with a name that is too short" in withEmptyDatabaseAndApiServices {
     services =>
       Post("/billing/v2",
-           CreateRawlsBillingProjectFullRequest(RawlsBillingProjectName("short"),
-                                                services.gcsDAO.accessibleBillingAccountName,
+        CreateRawlsV2BillingProjectFullRequest(RawlsBillingProjectName("short"),
```

Comment:
Wait, this was a V2 test that was calling the wrong API??

Reply:
It was calling the right API, but with the wrong payload. The payloads are nearly identical so the test still worked